Source/Diagnostics/ReducedDiags/MultiReducedDiags.cpp

```diff
@@ -0,0 +1,101 @@
+#include "MultiReducedDiags.H"
+#include "AMReX_ParmParse.H"
+#include "AMReX_Utility.H"
+#include <fstream>
+
+/// constructor
+MultiReducedDiags::MultiReducedDiags ()
+{
+
+    /// read reduced diags flag, names, and path
+    amrex::ParmParse pp("warpx");
+    pp.query("plot_reduced_diags", m_plot_rd);
+    if ( m_plot_rd == 0 ) { return; }
+    pp.getarr("reduced_diags_names", m_rd_names);
+    pp.query("reduced_diags_path", m_path);
+
+    /// creater folder
+    if (!amrex::UtilCreateDirectory(m_path, 0755))
+    { amrex::CreateDirectoryFailed(m_path); }
+
+    /// resize
+    m_multi_rd.resize(m_rd_names.size());
+
+    /// loop over all reduced diags
+    for (int i_rd = 0; i_rd < m_rd_names.size(); ++i_rd)
+    {
+
+        amrex::ParmParse pp(m_rd_names[i_rd]);
+
+        /// read reduced diags type
+        std::string rd_type;
+        pp.query("type", rd_type);
+
+        /// replace old and open output file
```

Comment:
What does "replace old" mean here?

Reply:
It means, if an old output file exits, replace it using `std::ios::trunc`, since later the file will be open as `std::ofstream::app`.